games/static/games/proto.js

```diff
@@ -35,6 +58,10 @@ class Game {
   start() {
     let gameId = 0;
     let timerId = 0;
+    let scoreId = 0;
+    scoreId = setInterval(() => {
```

Comment:
why do we need this?
shouldnt the score get updated whenever we hit alien or something ?

Reply:
I tried that way but working throgh the rocket function where the alien gets hit kept giving different results for the same test (shoot all the aliens and see the score is 120). So i did it through the alien movement instead so then i got constant results but it was updating only when the aliens was moving so i did this for immediate score display